WordPress/Sniffs/Theme/DiscouragedIframeSniff.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+/**
+ * Check for use of iframe. Often used for malicious code.
+ *
+ * @link    https://make.wordpress.org/themes/handbook/review/required/#core-functionality-and-features
+ *
+ * @package WPCS\WordPressCodingStandards
+ *
+ * @since   0.xx.0
+ */
+class WordPress_Sniffs_Theme_DiscouragedIframeSniff implements PHP_CodeSniffer_Sniff {
+
+	/**
+	 * The regex to catch the blacklisted attributes.
+	 *
+	 * @var string
+	 */
+	protected $iframe_regex;
+
+	/**
+	 * Returns an array of tokens this test wants to listen for.
+	 *
+	 * @return array
+	 */
+	public function register() {
+		$this->iframe_regex = '/<(iframe)[^>]*>/';
+
+		$tokens   = PHP_CodeSniffer_Tokens::$stringTokens;
+		$tokens[] = T_INLINE_HTML;
+
+		return $tokens;
+	}
+
+	/**
+	 * Processes this test, when one of its tokens is encountered.
+	 *
+	 * @param PHP_CodeSniffer_File $phpcsFile The file being scanned.
+	 * @param int                  $stackPtr  The position of the current token
+	 *                                        in the stack passed in $tokens.
+	 *
+	 * @return void
+	 */
+	public function process( PHP_CodeSniffer_File $phpcsFile, $stackPtr ) {
+		$tokens = $phpcsFile->getTokens();
+		$token  = $tokens[ $stackPtr ];
+
+		if ( preg_match( $this->iframe_regex, $token['content'] ) > 0 ) {
+			$phpcsFile->addWarning( 'iframes are sometimes used to load unwanted adverts and code on your site.' , $stackPtr, 'DiscouragedIframe' );
```

Comment:
`code` in that phrase, includes virusses and malware being downloaded in the background.
Are you sure you want reviewers to get virus ridden computers by checking the iframe content ? :heart_eyes: